Update README.md with a note about API URL for different regions (#283)
DataDog/datadog-cloudformation-resources: README.md

    }
  }
  ```

> Note: Provide `ApiURL` if using account other than US (defaults to https://api.datadoghq.com, use e.g. https://api.datadoghq.eu for EU accounts).

Reviewer: ```suggestion
   Specify the `ApiURL` if you are using a non-US account (defaults to `https://api.datadoghq.com`). For example, use `https://api.datadoghq.eu` for an EU account.
```
